user_guide_src/source/general/common_functions.rst

```diff
@@ -334,6 +334,10 @@ Miscellaneous Functions
     .. literalinclude:: common_functions/005.php
         :lines: 2-
 
+    .. note:: Usually this method will not clean up the fragment in the URL.
```

Comment:
I think this is not accurate because CI4 does nothing to a fragment. In fact, the fragment does not reach the server at all. The browser just keeps the fragment.

Reply:
I am agree with you, I will remove this line.

Reply:
I have been removed.